2021-09-09 14:20:31,483 INFO    MainThread:25788 [internal.py:wandb_internal():86] W&B internal server running at pid: 25788, started at: 2021-09-09 14:20:31.473506
2021-09-09 14:20:31,491 INFO    WriterThread:25788 [datastore.py:open_for_write():77] open: C:\Programming\rl_cooperation\Pygame_RL_cooperation\Algorithm\DQN_SpaceCannons_PER\wandb\offline-run-20210909_142029-uoq4ru75\run-uoq4ru75.wandb
2021-09-09 14:20:31,492 DEBUG   HandlerThread:25788 [handler.py:handle_request():126] handle_request: run_start
2021-09-09 14:20:31,512 DEBUG   HandlerThread:25788 [meta.py:__init__():40] meta init
2021-09-09 14:20:31,513 DEBUG   HandlerThread:25788 [meta.py:__init__():54] meta init done
2021-09-09 14:20:31,513 DEBUG   HandlerThread:25788 [meta.py:probe():211] probe
2021-09-09 14:20:31,580 DEBUG   HandlerThread:25788 [git.py:repo():33] git repository is invalid
2021-09-09 14:20:31,581 DEBUG   HandlerThread:25788 [meta.py:_save_pip():58] save pip
2021-09-09 14:20:31,582 DEBUG   HandlerThread:25788 [meta.py:_save_pip():72] save pip done
2021-09-09 14:20:31,582 DEBUG   HandlerThread:25788 [meta.py:probe():249] probe done
